mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -298,6 +294,18 @@
     <string name="browser_menu_tools">Tools</string>
     <!-- Content description (not visible, for screen readers etc.): Back button for all menu redesign sub-menu -->
     <string name="browser_menu_back_button_content_description">Back to main menu</string>
+    <!-- Content description (not visible, for screen readers etc.) for bottom sheet handlebar. The placeholder will be updated by, for example, "Main menu". -->
+    <string name="browser_menu_handlebar_content_description">Close %1$s sheet</string>
```

Comment:
@boek After giving this some more thought, it's improbable that some languages would have to change gender since the word "sheet" is the noun and remains this way throughout current existing strings.
However, it's still highly recommendable to have individual strings in this specific case (and since we're only talking about a few strings). For example, weird capitalization might happen when "stitching" these strings back together (see how "Main menu" and "Extensions menu" strings are both capitalized, but they will end up replacing the placeholder in the middle of the `browser_menu_handlebar_content_description` string, which will result in awkward looking strings).